examples/quickstart/main.rs

```diff
@@ -0,0 +1,124 @@
+                tvl_filter.clone(),
+                Some(balancer_pool_filter),
+            )
+            .auth_key(Some(tycho_api_key.clone()))
+            .set_tokens(all_tokens.clone())
+            .await
+            .build()
+            .await
+            .expect("Failed building protocol stream");
+
+        while let Some(msg) = protocol_stream.next().await {
+            tick_tx
+                .send(msg.unwrap())
+                .await
+                .expect("Sending tick failed!");
+        }
+        anyhow::Result::Ok(())
+    });
+
+    // Consumer task: Process messages and perform calculations
+    let consumer_handle = task::spawn(async move {
+        let mut pairs: HashMap<String, Vec<Token>> = HashMap::new();
+        while let Some(message) = tick_rx.recv().await {
+            println!(
+                "==================== Received block {:?} ====================",
+                message.block_number
+            );
+            for (id, comp) in message.new_pairs.iter() {
+                pairs
+                    .entry(id.clone())
+                    .or_insert_with(|| comp.tokens.clone());
+            }
+            if message.states.is_empty() {
+                println!("No pools were updated this block");
+                continue;
+            }
+            println!("Using only pools that were updated this block...");
+            for (id, state) in message.states.iter().take(10) {
+                if let Some(tokens) = pairs.get(id) {
+                    let formatted_token_str =
+                        format!("{:}/{:}", &tokens[0].symbol, &tokens[1].symbol);
+                    println!(
+                        "Calculations for pool {:?} with tokens {:?}",
+                        id, formatted_token_str
+                    );
+                    state
+                        .spot_price(&tokens[0], &tokens[1])
+                        .map(|price| println!("Spot price {:?}: {:?}", price, formatted_token_str))
+                        .map_err(|e| {
+                            eprintln!("Error calculating spot price for Pool {:?}: {:?}", id, e)
+                        })
+                        .ok();
+                    let amount_in =
+                        BigUint::from(1u32) * BigUint::from(10u32).pow(tokens[0].decimals as u32);
+                    state
+                        .get_amount_out(amount_in, &tokens[0], &tokens[1])
+                        .map(|result| {
+                            println!(
+                                "Amount out for trading 1 {:?} -> {:?}: {:?} (takes {:?} gas)",
+                                &tokens[0].symbol, &tokens[1].symbol, result.amount, result.gas
+                            )
+                        })
+                        .map_err(|e| {
+                            eprintln!("Error calculating amount out for Pool {:?}: {:?}", id, e)
+                        })
+                        .ok();
```

Comment:
Not needed I think? This result isn't used

Reply:
if I don't put it, I need to add a `let _ =` in the beginning. I prefer the `ok()`